client/header/index.js

```diff
@@ -30,6 +31,25 @@ export const Header = ( { sections, isEmbedded = false, query } ) => {
 		'is-scrolled': isScrolled,
 	} );
 
+	const updateBodyMargin = () => {
+		const wpBody = document.querySelector( '#wpbody' );
+		if ( ! wpBody ) {
+			return;
+		}
+
+		wpBody.style.marginTop = `${ headerElement.current.offsetHeight }px`;
+	};
+
+	useLayoutEffect( () => {
+		updateBodyMargin();
+		window.addEventListener( 'resize', updateBodyMargin );
```

Comment:
I wonder if we could benefit from some kind of debounce here on the resize event?

Reply:
Nice, I like this idea.  Will add it in.

Reply:
Added this in https://github.com/woocommerce/woocommerce-admin/pull/6247/commits/52c0a128eb8a868deef21a8fac0cff6ae884cb1f - note I did not use lodash's `debounce` to avoid using `lodash` in case we remove this dependency in the future.